packages/web-sdk/src/instrumentations/performance/performanceUtils.ts

```diff
@@ -119,10 +120,22 @@ export function calculateFaroNavigationTiming(navigationEntryRaw: PerformanceNav
 
     type: type,
 
-    ...calculateFaroResourceTiming(navigationEntryRaw),
+    ...createFaroResourceTiming(navigationEntryRaw),
   };
 }
 
+function getDocumentParsingTime(domInteractive: number): string {
+  let parserStart;
+  if (performance.timing?.domLoading != null) {
+    // the browser is about to start parsing the first received bytes of the HTML document.
+    // This property is deprecated but there isn't a really good alternative atm.
+    // For now we stick with domLoading and keep researching a better alternative.
+    parserStart = performance.timing.domLoading - performance.timeOrigin;
+  }
+
+  return toFaroPerformanceTimingString(parserStart ? domInteractive - parserStart : undefined);
```

Comment:
Q: Is there a chance that the parser start is greater than the domInteractive?

Reply:
I think this is meant to return null so that the `toFaroPerformanceTimingString` returns `unknown`. Maybe cover it with a testcase

Reply:
Q: Is there a chance that the parser start is greater than the domInteractive?
No (at least what I've read in the docs)